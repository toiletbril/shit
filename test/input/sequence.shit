echo 1; false; echo 2; false; echo 3;
echo 1 && echo 2 || echo 3;
false && echo 1 || false && echo 2 || true && echo 3;
