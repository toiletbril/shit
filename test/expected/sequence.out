[CompoundList]
  [CompoundListCondition, None]
    [SimpleCommand "echo 1"]
  [CompoundListCondition, None]
    [SimpleCommand "false"]
  [CompoundListCondition, None]
    [SimpleCommand "echo 2"]
  [CompoundListCondition, None]
    [SimpleCommand "false"]
  [CompoundListCondition, None]
    [SimpleCommand "echo 3"]
  [CompoundListCondition, None]
    [SimpleCommand "echo 1"]
  [CompoundListCondition, &&]
    [SimpleCommand "echo 2"]
  [CompoundListCondition, ||]
    [SimpleCommand "echo 3"]
  [CompoundListCondition, None]
    [SimpleCommand "false"]
  [CompoundListCondition, &&]
    [SimpleCommand "echo 1"]
  [CompoundListCondition, ||]
    [SimpleCommand "false"]
  [CompoundListCondition, &&]
    [SimpleCommand "echo 2"]
  [CompoundListCondition, ||]
    [SimpleCommand "true"]
  [CompoundListCondition, &&]
    [SimpleCommand "echo 3"]
[Escape Map
  
]
1
2
3
1
2
3
[Code 0]
